docs/runbook.md

```diff
@@ -0,0 +1,245 @@
+your target amount on that trend.  This example uses a nice round `1.0`
+billion DC.
+
+Convert HNT to DC via prevailing exchange rate from the
+[price oracle](https://api.helium.io/v1/oracle/prices/current).
+
+Optionally, get
+[predicted HNT Oracle Prices](https://api.helium.io/v1/oracle/predictions).
+"If no predictions are returned, the current HNT Oracle Price is valid for at
+least 1 hour."
+
+Calculate `$target_dc / $oracle_price / 1000` or as Lisp expression for
+target of 1.0 billion DC with HNT price of 2425000000 resulting in 412:
+
+```lisp
+(let ((current-hnt-price (/ 2425000000 1000))
+      (target-dc (* 1.0 1000 1000 1000)))
+  (floor (/ target-dc current-hnt-price)))
+```
+
+Use the resulting value as the `amount`:
+
+```bash
+helium-wallet burn --amount 412 --payee 123abc... --commit
+```
+
+Use the [Helium Wallet CLI](https://github.com/helium/helium-wallet-rs/)
+or its equivalent.
+
+## Snapshots
+
+Snapshots can be fetched by URL but as of early 2022 are approaching 300MiB.
+
+Fetch its metadata:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/latest-snap.json
+```
+
+The resulting JSON contains `height`, which gets used for file name of
+snapshot to download.
+
+```json
+{
+    "compressed_hash": "SGflG0oH0TTbairR3JCbtXoGJMhl6pYW4dHExmydRIc",
+    "compressed_size": 262995691,
+    "file_hash": "k0Pp00Y2wIeDagvzFMz66RAbD73e2y8fisCibP2Au2M",
+    "file_size": 427285643,
+    "hash": "3MmAnIgap0AMXL8n9eixxvnN8U8Qwtc3X_N6XXY1oR0",
+    "height": 1279441
+}
+```
+
+Craft the new URL based upon that height:
+
+```bash
+wget -N https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+```
+
+However, those are **generated** from the chain approximately **every 11 hours**.
+
+Therefore, compare chain heights or file timestamps.
+
+```bash
+curl https://api.helium.io/v1/blocks/height
+```
+
+Or:
+
+```bash
+curl --head https://snapshots.helium.wtf/mainnet/snap-1279441.gz
+TZ=Z date
+```
+
+Scripted version:
+
+```bash
+wget -N \
+  $(curl https://snapshots.helium.wtf/mainnet/latest-snap.json | \
+    jq .height | \
+    sed 's%^\(.*\)$%https://snapshots.helium.wtf/mainnet/snap-\1.gz%)'
+```
+
+The base of that URL is specified by `blockchain` `->` `snap_source_base_url`
+within
+[`sys.config.src`](../config/sys.config.src)
+with the filename being then-current block height.
+
+Similarly, locally generated snapshots should be within the directory path
+specified by `blockchain` `->` `base_dir`.
+
+### Manual Snapshots
+
+Start on a server confirmed to be current with respect to blockchain height.
+
+Maybe start on a validator, which uses the `minor` command rather than
+`router` but otherwise works the same.
+
+```bash
+miner snapshot take latest.snap
+docker cp validator:/opt/miner/latest.snap /tmp/
+rsync -a /tmp/latest.snap  192.168.123.123:/var/data/latest.snap
+```
+
+Once copied to the intended server, load that snapshot:
+
+```bash
+router snapshot load /var/data/latest.snap
+```
+
+## Tracing A Device
```

Comment:
We might note that traces disable themselves after 240mins. 
Or if the server get's restarted.
If you need a device trace longer than that, the trace needs to be started in a different way.

Reply:
What is the alternate way of starting such a trace?

Reply:
It's semi-hardcoded into the trace functionality right now. 
But there's the settings
```
application:get_env(router, device_trace_timeout, 240)
```
that can be changed before a trace of a different length.

Reply:
Updated in 64b4652 along with the comment from `#blockchain` regarding the env var.